models/iaf_psc_alpha_multisynapse.h

```diff
@@ -158,6 +162,25 @@ class iaf_psc_alpha_multisynapse : public Archiving_Node
    */
   struct State_
   {
+
+    /**
+     * Enumeration identifying elements in state vector State_::y_.
+     * This enum identifies the elements of the vector. The last element
+     * of this (I_SYN, ) will be repeated n times at the end of the
+     * state vector State_::y with n being the number of synapses.
+     * As a first step this vector will only be used for logging.
+     */
+    enum StateVecElems
+    {
+      V_M = 0,
+      I,     // 1
+      I_SYN, // 2
+      STATE_VECTOR_MIN_SIZE
+    };
+
+    static const size_t NUMBER_OF_FIXED_STATES_ELEMENTS = 2; // V_M, I
```

Comment:
Instead of initializing with the "magic" value 2 here, initialize with `I_SYN` to guarantee consistency. Alternatively, you could replace `I_SYN` in the enum above by `NUM_FIXED_STATE_ELEMENTS`; then you would not need the additional constant here. Maybe one could then also drop the `STATE_VECTOR_MIN_SIZE`.